geonode_mapstore_client/client/js/utils/PluginsContextUtils.js

```diff
@@ -14,12 +14,33 @@ import {
 } from '@js/utils/ResourceUtils';
 import get from 'lodash/get';
 
+function getUserResourceName(user) {
+    return user?.first_name !== '' && user?.last_name !== ''
+        ? `${user?.first_name} ${user?.last_name}`
+        : user?.username;
+}
+
+function userListToCsv(users = []) {
+    if (!users) {
+        return "";
+    }
```

Comment:
Is it possible this case? I'm seeing the the argument users is providing a default empty array so probably this return with an empty string is not really needed.
Maybe is needed if users argument is `null`, `""` or `false` but not sure what the resources endpoint could return for `poc` field

Reply:
You're right. I guess, this `return` is an artifact from a previous implementation version which did not return an array. I would suggest to return `[]` in such cases. What do you think?

Reply:
@ridoo yes, I think the `[]` is fine, thanks